frontend/src/metabase/query_builder/components/QuestionActivityTimeline.jsx

```diff
@@ -101,39 +77,42 @@ export function QuestionActivityTimeline({
     const revisionEvents = getRevisionEventsForTimeline(revisions, {
       currentUser,
       canWrite,
+    }).map(revisionEvent => {
+      if (revisionEvent.isRevertable) {
+        const { title, revision } = revisionEvent;
+        const newTitle = (
+          <>
+            {title}
+            <Tooltip tooltip={t`Revert`} placement="bottom">
+              <RevertButton
+                icon="revert"
+                onlyIcon
+                borderless
+                onClick={() => revertToRevision(revision)}
+              />
+            </Tooltip>
+          </>
+        );
+        return { ...revisionEvent, title: newTitle };
+      } else {
+        return revisionEvent;
+      }
     });
-    return [...revisionEvents, ...moderationEvents];
-  }, [canWrite, moderationReviews, revisions, usersById, currentUser]);
 
-  const onDrawerStateChange = state => {
-    if (state === DRAWER_STATES.open) {
-      onOpenQuestionHistory();
-    } else {
-      onCloseQuestionHistory();
-    }
-  };
+    return [...revisionEvents, ...moderationEvents];
+  }, [
+    canWrite,
+    moderationReviews,
+    revisions,
+    usersById,
+    currentUser,
+    revertToRevision,
+  ]);
 
   return (
-    <DrawerSection
-      header={t`History`}
-      state={drawerState}
-      onStateChange={onDrawerStateChange}
-    >
-      <Timeline
-        items={events}
-        data-testid="saved-question-history-list"
-        renderFooter={item => {
-          const { isRevertable, revision } = item;
-          if (isRevertable) {
-            return (
-              <RevisionEventFooter
-                revision={revision}
-                onRevisionClick={revertToRevision}
-              />
-            );
-          }
-        }}
-      />
-    </DrawerSection>
+    <div>
+      <Header>History</Header>
```

Comment:
Since the previous copy is translated, I assume this one should too.
```suggestion
      <Header>{t`History`}</Header>
```

Reply:
I almost made a whole PR without forgetting to translate something. So close lol